config/config.go

```diff
@@ -396,7 +396,7 @@ func (c *chainConfig) parseAndValidateSporks(ctx context.Context, result *Chain)
 			Version:   cfg.Version,
 		}
 
-		if spork.Version < 1 || spork.Version > 7 {
+		if spork.Version < 1 || spork.Version > 8 {
```

Comment:
Could you add a comment to the [`Spork.Version` field](https://github.com/onflow/rosetta/blob/dc9a3c9bc69e2ca2180c76f7a86d991977a36935/config/config.go#L115):
```go
// Version is a Rosetta-internal version number that tracks Rosetta compatibility with Flow network versions.
// This version number is incremented each time there is a Flow network upgrades which includes a breaking change for Rosetta.
// Not all Flow network upgrades cause breaking changes for Rosetta, so this version number is not incremented for every network upgrade.